tests/helperServer.js

```diff
@@ -138,6 +142,25 @@ module.exports = {
               callback(null);
             });
           },
+          function (callback) {
+            externalGrader.init(assessment, function (err) {
+              if (ERR(err, callback)) return;
+              callback(null);
+            });
+          },
```

Comment:
Note to self: refactor so that `externalGrader` doesn't need to take `assessment` as an argument. That module should just require `lib/assessment` itself.

Reply:
Done in 83740f6.